### PR TITLE
Further Job and Rank Changes

### PR DESCRIPTION
**Pull request by Andromeda-K22**
https://docs.google.com/document/d/1bTXZ3urA5KA56ccxpBVzeEv4lUPEXFPXn4WJBZu_vt8/edit

### CODE REVIEW

File: maps/torch/job/jobs.dm [hunk 1]
```diff
@@ -1,29 +1,29 @@
 /datum/map/torch
 	species_to_job_whitelist = list(
 		/datum/species/nabber = list(/datum/job/ai, /datum/job/cyborg, /datum/job/janitor, /datum/job/scientist_assistant, /datum/job/chemist,
-		/datum/job/roboticist, /datum/job/cargo_contractor, /datum/job/chef,   /datum/job/bartender),
+		/datum/job/roboticist,   /datum/job/chef,   /datum/job/bartender),
```
Comment (MalfunctioningAI) [inline maps/torch/job/jobs.dm]: Triple spacing?

**Comment by MalfunctioningAI** [comment]
## UNIT_TEST ##: [JOB: Shall have a HUD icon]: Nurse - Missing HUD icon: hudnurse
## UNIT_TEST ##: !!! FAILURE !!! [JOB: Shall have a HUD icon]: maps/torch/icons/assignment_hud.dmi - 0 failed sanity checks, 1 job with missing HUD icon.

**Comment by MalfunctioningAI** [comment]
```[JOB: Shall have a HUD icon]: Nurse - Missing HUD icon: hudnurse
[JOB: Shall have a HUD icon]: maps/torch/icons/assignment_hud.dmi - 0 failed sanity checks, 1 job with missing HUD icon.```

**Comment by MalfunctioningAI** [comment]
`[JOB: Shall have a HUD icon]: Nurse - Missing HUD icon: hudnurse`
`[JOB: Shall have a HUD icon]: maps/torch/icons/assignment_hud.dmi - 0 failed sanity checks, 1 job with missing HUD icon.`

File: maps/torch/job/jobs.dm (hunk 1 above)
Comment (Andromeda-K22) [reply maps/torch/job/jobs.dm]: will fix

**Comment by Andromeda-K22** [comment]
thank

**Comment by MalfunctioningAI** [comment]
When you are done, feel free to squash your commits. Doing this makes everything neat and tidy. (See #8 for an example.)

If you need help doing this, just ask. It is always a good thing to know.

**Pull request merged**